admin.tf

```diff
@@ -137,3 +137,56 @@ resource "aws_elb" "admin-elb" {
         env                       = "shared"
     }
 }
+
+# EFS mount targets for `marathon-efs` backups
+
+resource "aws_security_group" "marathon-efs-admin-sg" {
```

Comment:
The security group + rule should live in storage.tf since they are are inbound rules for storage.